content/01.abstract.md

```diff
@@ -11,3 +11,21 @@ Publication with Manubot is continuous:
 When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
 Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers (e.g. DOIs, PubMed IDs, ISBNs, URLs), whose metadata is then retrieved and converted to a user-specified style.
 Manubot modernizes publishing to align with the ideals of open science by making it transparent, reproducible, immediate, versioned, collaborative, and free of charge.
+
+## Author summary
+
+Traditionally, scholarly manuscripts, such as this one you are reading, have been written in private.
+This worked because the research was also performed in private by a predefined team of collaborators.
+But now the internet has enabled realtime open science, where project communication occurs online in a public venue and anyone is able to contribute.
+But how can dispersed teams of online contributors jointly prepare manuscripts to describe their work?
+
+Existing tools fail to scale and struggle to enable iterative refinement of proposed changes.
+Therefore, we created a system for writing manuscripts on GitHub, called Manubot.
+Manubot adopts the workflow from open source software development, which has enabled hundreds of contributors to simultaneously develop complex codebases such as Python and Linux, and applies it to open collaborative writing.
+
+In addition to collaboration at scale, Manubot addresses other shortcomings of current publishing tools.
+Specifically, all changes to a manuscript are tracked, enabling transparency and better attribution of credit.
+Manubot automates as many tasks as possible, including bibliography creation.
+Users cite persistent identifiers, such as DOIs, and then Manubot retrieves and formats the metadata into references.
+Furthermore, every version of a manuscript is automatically published and preserved with a permalink.
+As such, Manubot manuscripts are like continuously updating preprints, where readers have ample opportunity to not only provide post-publication peer review but also to contribute improvements.
```

Comment:
Can you please clarify how they are like preprints?  We could refocus this to say:
> Manubot manuscripts can be continuously updated, where readers have ample opportunity to not only provide peer review but also to contribute improvements.

Reply:
I like the rephrasing. `where readers` sounds funny to me, but I'm not sure how I'd rephrase it. Maybe I'd say `whereby` to mean "...manuscripts can be updated by readers..." but that sounds archaic, so I'm okay with `where` in your version.

Reply:
We could use
> continuously updated, meaning readers...

or
> continuously updated.  Readers...

I'm a fan of short, simple sentences in the summary paragraph.

Reply:
Take a look at the last sentence in ddd5222b987a04a9740d577130024e3cee7a2964 (still long) and suggest any improvements you see fit. I want to tie this in with preprints, since I think that is an important topic these days. Primarily, how manubot takes preprints to the next level by:

1. making them more customizeable, automated, and versioned
2. making them more open from early stages when feedback is most actionable

Reply:
I think you could expand and split into two sentences. Something like...

> As such, Manubot produces an ecosystem of living preprint. 
> Readers of in-progress and complete manuscripts have ample opportunity to not only provide public peer review but also to contribute improvements, before and after publication.

What do you think?

Reply:
This idea is a good closing.  Can we push more on how it takes preprints to the next level, echoing @dhimmel's points and phrasing above?  That may not be implied strongly enough by referring to "living" preprints?  Maybe

> As such, Manubot extends the benefits of preprints.
Manuscript readers have ample opportunity to not only provide public peer review but also to contribute improvements, before and after publication.

The latest commits resolved everything else except the word count, which I didn't check.

Reply:
Regarding word count, we are probably good (right around 200).

@agitter I like your phrasing. What do you think about keeping your second sentence, but replacing your first sentence with:

+ As such, Manubot is a suitable foundation for next-generation preprints.
+ As such, Manubot is a suitable foundation for the next generation of preprints.

No strong opinions, just wanted to see if this was appealing. In some other places we use the "next-generation manuscripts" tagline, so it could be good branding.

Reply:
I prefer the first `next-generation preprints` phrasing.   It's a bold statement, but I'm okay with that.  We want to present a strong vision for the future.

Reply:
Okay I think we should be good as of https://github.com/greenelab/meta-review/pull/220/commits/79f872040d86d94b481457ea872c515453584843. Let me know if I missed anything.